src/common/location/__tests__/Location.test.tsx

```diff
@@ -0,0 +1,56 @@
+import { render } from '@testing-library/react';
+
+import { useMediaQuery } from '@chakra-ui/react';
+
+import Location from '../Location';
+
+jest.mock('@chakra-ui/react', () => ({
+  ...jest.requireActual('@chakra-ui/react'),
+  useMediaQuery: jest.fn().mockReturnValue(false),
+}));
+
+describe('Location Component', () => {
+  it('should render the short form on mobile', () => {
+    (useMediaQuery as unknown as jest.Mock).mockReturnValue([true]);
```

Comment:
Just wondering why this type casting is required? Not dogging it cause I know testing can be a pain sometimes just curious.

Reply:
yeah I was having trouble mocking it so tried a bunch of different things but just tried again and the `as unknown` is not necessary, just casting it as jest Mock works as I need in that type to be able to mockReturnValue! I've simplified the type casting on it